src/controllers/v1/attachment/index.ts

```diff
@@ -87,23 +86,25 @@ export class attachment extends Controller {
 
   @Get('/')
   @SuccessResponse(200, 'returns all attachment')
-  public async get(@Query() updated_since?: DATE): Promise<Attachment[]> {
-    this.log.debug('retrieving all attachment')
-
-    const attachments: AttachmentRow[] = await this.db.get(
-      'attachment',
-      updated_since ? { created_at: parseDateParam(updated_since) } : {}
-    )
-    return attachments.map(({ ipfs_hash, ...rest }): Attachment => ({ ...rest, createdAt: rest.created_at }))
+  public async getAll(@Query() createdAt?: DATE): Promise<Attachment.Response> {
+    this.log.debug('retrieving all attachments')
+    const where: Record<string, number | string | DATE> = {}
+    if (createdAt) where.created_at = createdAt
+
+    return {
+      message: 'ok',
+      attachments: await this.db.get('attachment', where),
+    }
   }
 
   @Post('/')
   @SuccessResponse(201, 'attachment has been created')
   @Response<ValidateError>(422, 'Validation Failed')
+  @Response<BadRequest>(400, 'Invalid request')
   public async create(
     @Request() req: express.Request,
     @UploadedFile() file?: Express.Multer.File
-  ): Promise<Attachment> {
+  ): Promise<Attachment.Response | string> {
```

Comment:
why the `| string`

Reply:
this should be as part of Attachment.Response, because of Errors/Exceptions